src/rpc/client.cpp

```diff
@@ -219,12 +219,13 @@ UniValue RPCConvertValues(const std::string &strMethod, const std::vector<std::s
     for (unsigned int idx = 0; idx < strParams.size(); idx++) {
         const std::string& strVal = strParams[idx];
 
-        if (!rpcCvtTable.convert(strMethod, idx)) {
+        UniValue json_value;
+        if (!rpcCvtTable.convert(strMethod, idx) || !json_value.read(strVal)) {
```

Comment:
Seems odd to have two inversions in a condition. Would be easier to write
```py
if convert() and read():
 # push back parsed value
else:
 # push back str

Reply:
Keep diff small? 👼  But agree.